src/components/MenuButton/MenuButton.md

```diff
@@ -6,13 +6,13 @@ Context menu should be used as a menu button where you want to provide options t
 
 MenuButton with default Icon, and all applicable children properties
 
-```js { "props": { "data-description": "with default icon and all menu item types" } }
+```js { "props": { "data-description": "with default icon and all menu item types", "data-action-states": "[{\"action\":\"none\"},{\"action\":\"click\",\"selector\":\".y-menu-button\"}]" } }
```

Comment:
can you do all of these interactive diffs in a separate PR? the change required to fix #327 worries me a bit, it would be nice to have the screenshots in place for the current code first. thanks!